enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/api/WebSocketAPI.java

```diff
@@ -119,13 +123,35 @@ public String init(Api api) {
                     APIProcessUtils.convertProtoEndpointSecurity(
                             api.getEndpointSecurity().getSandBoxSecurityInfo()));
         }
+        for (Resource res : api.getResourcesList()) {
+            for (Operation operation : res.getMethodsList()) {
+                ResourceConfig resConfig = RestAPI.buildResource(operation, res.getPath(), new HashMap<>());
+                resConfig.setTier(api.getTier());
+                resources.add(resConfig);
+            }
+        }
+
+        Map<String, SecuritySchemaConfig> securitySchemeDefinitions = new HashMap<>();
+
+        for (SecurityScheme securityScheme : api.getSecuritySchemeList()) {
+            if (securityScheme.getType() != null) {
+                String definitionName = securityScheme.getDefinitionName();
+                SecuritySchemaConfig securitySchemaConfig = new SecuritySchemaConfig();
+                securitySchemaConfig.setDefinitionName(definitionName);
+                securitySchemaConfig.setType(securityScheme.getType());
+                securitySchemaConfig.setName(securityScheme.getName());
+                securitySchemaConfig.setIn(securityScheme.getIn());
+                securitySchemeDefinitions.put(definitionName, securitySchemaConfig);
+            }
+        }
```

Comment:
We have the security schemes in line 88.

Reply:
fixed with https://github.com/wso2/product-microgateway/pull/2667/commits/5b61cc26f7560e106431afd9027b3066aeea76ad